opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/config/CustomizedHbciBanking.java

```diff
@@ -0,0 +1,76 @@
+package de.adorsys.opba.protocol.hbci.config;
+
+import de.adorsys.multibanking.domain.request.AbstractRequest;
+import de.adorsys.multibanking.domain.request.TransactionRequest;
+import de.adorsys.multibanking.domain.response.AbstractResponse;
+import de.adorsys.multibanking.domain.response.PaymentResponse;
+import de.adorsys.multibanking.domain.response.TransactionAuthorisationResponse;
+import de.adorsys.multibanking.domain.transaction.AbstractPayment;
+import de.adorsys.multibanking.domain.transaction.TransactionAuthorisation;
+import de.adorsys.multibanking.hbci.HbciBanking;
+import de.adorsys.multibanking.hbci.HbciBpdUpdCallback;
+import de.adorsys.multibanking.hbci.HbciCacheHandler;
+import de.adorsys.multibanking.hbci.HbciExceptionHandler;
+import de.adorsys.multibanking.hbci.job.ScaAwareJob;
+import de.adorsys.multibanking.hbci.job.SinglePaymentJob;
+import de.adorsys.multibanking.hbci.model.HbciConsent;
+import de.adorsys.opba.protocol.hbci.service.protocol.CustomizedOnlineBankingService;
+import lombok.SneakyThrows;
+import org.kapott.hbci.exceptions.HBCI_Exception;
+import org.kapott.hbci.manager.HBCIProduct;
+
+import java.lang.reflect.Method;
+
+import static de.adorsys.multibanking.domain.ScaStatus.FINALISED;
+
+public class CustomizedHbciBanking extends HbciBanking implements CustomizedOnlineBankingService {
```

Comment:
I think we need to add a comment why this one exists or drop it completely (as it is just to set e2e id as it didn't work maybe we can drop this)